core/src/com/po/fuck/view/classdrawers/Misc.java

```diff
@@ -0,0 +1,12 @@
+package com.po.fuck.view.classdrawers;
+
+import com.badlogic.gdx.graphics.Texture;
+import com.badlogic.gdx.graphics.g2d.Animation;
+import com.badlogic.gdx.graphics.g2d.TextureRegion;
+import com.badlogic.gdx.graphics.g2d.Sprite;
+
+public class Misc {
+    public static Animation<TextureRegion> getAnimation(String internalPath){
+        return new Animation<TextureRegion>(1f, new Sprite(new Texture(internalPath)));
```

Comment:
Why is this `1f` here ?

Reply:
It is a duration of each frame in Animation. If there is the only frame, this value does not really matter.

Reply:
Add a comment, that says that this is a dummy value. Try to pass `0` as it is more common value.